google-cloud-tools-plugin/testSrc/com/intellij/appengine/PicoContainerTestUtil.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intellij.appengine;
+
+import com.google.auto.value.AutoValue;
+
+import com.intellij.openapi.application.ApplicationManager;
+
```

Comment:
This is how the IntelliJ import optimizer is formatting it, but I agree, it shouldn't be like this. In fact, Google recently revised their style rules to say all imports should be in the same block: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Should I use this approach (static in one block, others in another block) on all new files?